src/colorizer/utils/url_utils.ts

```diff
@@ -29,6 +29,7 @@ import { numberToStringDecimal } from "./math_utils";
 // TODO: This file needs to be split up for easier reading and unit testing.
 // This could also be a great opportunity to reconsider how we store and manage state.
 
+export const URL_COLOR_RAMP_REVERSED_SUFFIX = "!";
```

Comment:
This constant got moved out of `UrlParam` because it's technically not its own param, just a single character used in one of the params.